### DISCUSSION TITLE
Welcome to venom Discussions!

### DISCUSSION THREAD

**Discussion by yesnault** [ovh/venom#344] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FAC-1312** [comment]
Hello everyone, I'm new on Venom I knew this community when I came to a new project in France, it's quite nice as solution for integration testing I love so much. 

Thank you.
